inference-contract/docs/notebooks/documents/overview.md

```diff
@@ -0,0 +1,61 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.1
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Inference contracts using PDO #
+
+The Inference contract family is a suite of contracts that extend the
+capabilities of the [Exchange Contract Family ](../exchange/documents/overview.ipynb)
+in order to create a confidentiality preserving policy-wrapper around the
+usage of a machine learning (ML) model. At its core, the implementation
+contains a Token Object PDO contract that specifies and checks the
+policies to be followed while using the ML model for inferencing
+operations. In this implementation, the ML Model is assumed to be an
+[OpenVINO based IR](https://docs.openvino.ai/2023.2/openvino_ir.html)
+model. The model itself is deployed via an asset guardian service; the
```

Comment:
Do we have documentation explaining what an "asset guardian service" is? As above, it would be good to add a link here.